docs/getting-started/index.md

```diff
@@ -69,9 +71,20 @@ WSL may be having networking issues.  To resolve this, you can follow the steps
 ### Set Up a New Test Kit
 {:toc-skip}
 
-1. Clone the [Inferno Template](https://github.com/inferno-framework/inferno-template) repository. You can
-   either clone this repository directly, or click the green "Use this template"
-   button to create your own repository based on this one.
+There are two convenient methods to create a new Test Kit. The first is by cloning the 
+[Inferno Template](https://github.com/inferno-framework/inferno-template) repository. This
+method ensures you will always be starting with the latest Test Kit Template. You can
+either clone this repository directly, or click the green "Use this template" button in GitHub
+to create your own repository based on the Inferno Template.
+
+The second method is to use the Inferno Command Line Interface. After installing
+the `inferno_core` gem, run `inferno new TestKitName` to create a new Test Kit
+Template in a directory named `test-kit-name`. See how to [create a new Test
+Kit](/docs/getting-started/inferno-cli.html#creating-a-new-test-kit) using the Inferno CLI for more
+information and options.
+
+1. Create your Test Kit by cloning the [Inferno Template](https://github.com/inferno-framework/inferno-template)
+   or by using the `inferno new` command.
 1. Run `bundle install` to install dependencies.
 1. Run `bundle exec inferno migrate` to set up the database.
```

Comment:
These could also be code blocks instead of inline-code for easy copy-paste, but if they're any plans for a quickstart guide that would be the place to put these instead.